### PR TITLE
feat: add presence for skyra.pw

### PR DESCRIPTION
**Pull request by Favna**
I'd like to add a Presence for [skyra.pw](https://skyra.pw/), the website and dashboard for [Skyra](https://github.com/skyra-project/skyra)

Before a remark is made about the using of `data-premid` in the query-selectors, I'm a co-owner of Skyra and added those attributes to the sourcecode of the website myself, so it's a perfectly safe selector to use. 

Screenshots in collapsed view:

<details><summary>Click here to view screenshots</summary>

image
image
image
image

</details

### CODE REVIEW

**Comment by doomlerd** [comment]
Hello @Favna, your PR is not merged yet because we're making some changes in favor of quality assurance which will require your PR to be re-made, we'll inform you what to do next ( to comply with the new changes ), I'll keep this on hold now until we finish.

For reference : #1248 ( rebase #1258 )

File: Skyra/presence.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+var title: Element;
+
+presence.on("UpdateData", async () => {
+  var presenceData: presenceData = {
+    largeImageKey: "skyra",
+  };
```
Comment (Bas950) [inline Skyra/presence.ts]: remove comma

File: Skyra/presence.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+var presence = new Presence({
+  clientId: "266624760782258186",
+});
```
Comment (Bas950) [inline Skyra/presence.ts]: remove comma

**Comment by doomlerd** [comment]
Sorry for the wait, now that the repo is unlocked you have to recreate the PR with these changes :
apply the changes @Bas950 requested, format your code with either spaces or tabs and using only 2 or 4 in tab width, and run the following compiler command to recompile your files : 
`tsc --noFallthroughCasesInSwitch --noUnusedLocals --noUnusedParameters -t "ES2020"`
then recreate the PR for the new checks to start.

File: Skyra/presence.ts (hunk 2 above)
Comment (Favna) [reply Skyra/presence.ts]: Yeah sure. Just as an aside though, the commas were added by Prettier. If you do not want trailing commas you may want to change `es5` to `none` in the `.prettierrc.json` file, right now it's `es5` which adds trailing commas where it's valid to have them as per the ES5 spec.

For more info see Prettier options: https://prettier.io/docs/en/options.html#trailing-commas

**Pull request closed without merging**